chat_api_sample/src/main/java/com/zopim/sample/chatapi/chat/ChatView.java

```diff
@@ -169,7 +169,12 @@ public void onClick(final View view) {
     private void initChatLogRecycler() {
         final RecyclerView recyclerView = views.chatRecycler;
 
-        final RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(context);
+        final RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(context) {
+            @Override
+            public boolean supportsPredictiveItemAnimations() {
+                return false;
```

Comment:
What's this all about?

Reply:
There was a change in the latest release of Chat SDK that results (I think) in some non-display items being removed from the recyclerviews adapter source. This wasn't handled properly in this sample app when it was written causing `IndexOutOfBoundsException` being thrown. This override stops the issue but isn't the best solution :/

Reply:
Replaced it with a logic fix rather than UI fix, cheers Rich!